python/lsst/cp/verify/verifyDefects.py

```diff
@@ -0,0 +1,142 @@
+                                  'STDEV': 'STDEVCLIP',
+                                  'MIN': 'MIN',
+                                  'MAX': 'MAX', }
+        self.unmaskedImageStatKeywords = {'UNMASKED_MIN': 'MIN',  # noqa F821
+                                          'UNMASKED_MAX': 'MAX',
+                                          'UNMASKED_OUTLIERS': 'NCLIPPED', }
+
+
+class CpVerifyDefectsTask(CpVerifyStatsTask):
+    """
+    """
+    ConfigClass = CpVerifyDefectsConfig
+    _DefaultName = 'cpVerifyDefects'
+
+    def imageStatistics(self, exposure, statControl):
+        """Measure additional defect statistics.
+
+        This calls the parent class method first, then adds additional
+        measurements.
+
+        Parameters
+        ----------
+        exposure : `lsst.afw.image.Exposure`
+            Exposure containing the ISR processed data to measure.
+        statControl : `lsst.afw.math.StatControl`
+            Statistics control object with the parameters defined by
+            the config.
+
+        Returns
+        -------
+        outputStats : `dict` [`str`, `dict` [`str`, scalar]]
+            A dictionary indexed by the amplifier name, containing
+            dictionaries of the statistics measured and their values.
+        """
+        outputStats = super().imageStatistics(exposure, statControl)
+
+        # Is this a useful test?  It saves having to do chi^2 fits,
+        # which are going to be biased by the bulk of points.
+        for amp in exposure.getDetector():
+            ampName = amp.getName()
+            ampExp = exposure.Factory(exposure, amp.getBBox())
+
+            normImage = ampExp.getImage()
+            normArray = normImage.getArray()
+
+            normArray -= outputStats[ampName]['MEDIAN']
+            normArray /= outputStats[ampName]['STDEV']
+
+            probability = scipy.stats.norm.pdf(normArray)
+            trials = np.prod(probability.shape)
+            outliers = np.where(probability < 1.0 / trials, 1.0, 0.0)
+            outputStats[ampName]['STAT_OUTLIERS'] = np.sum(outliers)
+
+        return outputStats
+
+    def verify(self, exposure, statisticsDictionary):
+        """Verify if the measured statistics meet the verification criteria.
+
+        Parameters
+        ----------
+        statisticsDictionary : `dict` [`str`, `dict` [`str`, scalar]],
+            Dictionary of measured statistics.
+
+        Returns
+        -------
+        outputStats : `dict` [`str`, `dict` [`str`, `bool`]]
+            A dictionary indexed by the amplifier name, containing
+            dictionaries of the verification criteria.
+        success : `bool`
+            A boolean indicating if all tests have passed.
+        """
+        ampStats = statisticsDictionary['AMP']
+        verifyStats = dict()
+        success = True
+        for ampName, stats in ampStats.items():
+            verify = dict()
+
+            # These are not defined yet.
```

Comment:
Not defined in DMTN-101.  This package (as it will say in the README.md once I push change) will implement all of the tests defined there, but the defect section is currently unwritten.  I wanted this first pass at the package to be reviewed and merged before sitting down to define everything that has been left undone.  I've updated the comment to point to the DMTN-101 definition.